packages/base/src/3dview/mainview.tsx

```diff
@@ -1065,6 +1065,46 @@ export class MainView extends React.Component<IProps, IStates> {
         );
       }
 
+      if (this._camera instanceof THREE.OrthographicCamera) {
+        const boundingSphere = new THREE.Sphere();
+        this._boundingGroup.getBoundingSphere(boundingSphere);
+        const center = boundingSphere.center;
+        const radius = boundingSphere.radius;
+
+        const aspect =
+          this._renderer.domElement.clientWidth /
+          this._renderer.domElement.clientHeight;
+        const frustumSize = radius * 2;
+
+        // Position the camera at an angle away from the center
+        const offset = radius * 2;
+        this._camera.position.set(
+          center.x + offset,
+          center.y + offset,
+          center.z + offset
+        );
+        this._camera.lookAt(center);
+
+        // Update orthographic frustum
+        this._camera.left = (-frustumSize * aspect) / 2;
+        this._camera.right = (frustumSize * aspect) / 2;
+        this._camera.top = frustumSize / 2;
+        this._camera.bottom = -frustumSize / 2;
+
+        // Set near/far conservatively
+        this._camera.near = 0.01;
```

Comment:
Since 0 is a valid value for the orthographic camera

```suggestion
        this._camera.near = 0;
```